grpc/orderbook.go

```diff
@@ -23,6 +24,9 @@ func NewOrderbookClient() orderbook.Client {
 
 // OpenOrder implements the orderbook.Client interface.
 func (client *orderbookClient) OpenOrder(ctx context.Context, multiAddr identity.MultiAddress, orderFragment order.EncryptedFragment) error {
+	if orderFragment.IsEmpty() {
+		return fmt.Errorf("cannot open order for empty order fragment")
```

Comment:
More consider error descriptions. If the error does not need to be parameterised, then declare the error as a value and use it.

```go
var ErrOrderFragmentIsNil = errors.New("order fragment is nil")
```